src/plugins/es.json

```diff
@@ -0,0 +1,205 @@
+    "proceduralFairnessImplementation4": "Todos los puntos clave de las decisiones están relacionados con la legislación, la política o el procedimiento pertinentes.",
+    "proceduralFairnessImplementation5": "Se ha desarrollado un log de cambios para detallar todas las modificaciones realizadas en el modelo y en el sistema.",
+    "proceduralFairnessImplementation6": "La pista de auditoría identifica qué versión del sistema se utilizó para cada decisión.",
+    "proceduralFairnessImplementation7": "La pista de auditoría del sistema indica todos los puntos de decisión realizados por el sistema.",
+    "proceduralFairnessImplementation8": "El seguimiento de auditoría generado por el sistema puede utilizarse para generar notificaciones, incluida una declaración de motivos.",
+    "proceduralFairnessImplementation9": "El log del sistema muestra quiénes son los responsables de la toma de decisiones autorizados.",
+    "proceduralFairnessImplementation10": "El sistema puede presentar las razones de su decisión o recomendaciones cuando sea necesario.",
+    "proceduralFairnessImplementation11": "Existe un proceso para otorgar, rastrear y revocar el permiso de acceso al sistema.",
+    "proceduralFairnessImplementation12": "Existe un mecanismo para captar la retro alimentación de los usuarios de los sistemas.",
+    "proceduralFairnessImplementation13": "Existe un proceso de recurso para los clientes que deseen impugnar la decisión.",
+    "proceduralFairnessImplementation14": "El sistema permite la anulación humana de las decisiones del sistema.",
+    "proceduralFairnessImplementation15": "Existe un proceso para registrar las instancias en las que se realizaron las sustituciones.",
+    "proceduralFairnessImplementation16": "La pista de auditoría del sistema incluye procesos de control de cambios para registrar las modificaciones en el funcionamiento o rendimiento del sistema.",
+    "proceduralFairnessImplementation17": "La Junta de Examen de la Arquitectura Empresarial del Gobierno del Canadá ha preparado y aprobado un caso conceptual.",
+    "privacyImplementation1": "",
+    "privacyImplementation2": "",
+     "requirements": {
+        "title": "Requisitos específicos para el nivel de impacto:",
+        "elements": [{
+                "title": "revisión por pares",
+                "elements": [{
+                    "text": "None"
+                }, {
+                    "text": "At least one of:\n\nQualified expert from a federal, provincial,territorial or municipal government institution\n\nQualified members of faculty of a post - secondary institution\n\nQualified researchers from a relevant non - governmental organization\n\nContracted third - party vendor with a related specialization\n\nPublishing specifications of the Automated Decision System in a peer - reviewed journal\n\nA data and automation advisory board specified by Treasury Board Secretariat"
```

Comment:
Just wondering if you left this section in English because it's specific to our own Policies or if it's an oversight?